### PR TITLE
fix: store data of new last run immediately when launching a new run

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

File: src/state/sagas/scenario/LaunchScenario/LaunchScenario.js [hunk 1]
```diff
@@ -29,10 +29,17 @@ export function* launchScenario(action) {
     });
 
     // Launch scenario if parameters update succeeded
-    yield call(Api.ScenarioRuns.runScenario, organizationId, workspaceId, scenarioId);
+    const res = yield call(Api.ScenarioRuns.runScenario, organizationId, workspaceId, scenarioId);
```
Comment (esasova) [inline src/state/sagas/scenario/LaunchScenario/LaunchScenario.js:32]: **annoying voice**: Could we call this constant `response` to be coherent with other sagas ? :sweat_smile:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/state/sagas/scenario/LaunchScenario/LaunchScenario.js (hunk 1 above)
Comment (csm-thu) [reply src/state/sagas/scenario/LaunchScenario/LaunchScenario.js]: Done :heavy_check_mark:

**Review by ardvi — APPROVED** [review]
LGTM :+1:

**Comment by csm-thu** [comment]
Manually merged in _main_

**Pull request closed without merging**